cypress-io/cypress
# fix: disable all aspects of protocol in non chromium browsers

### Additional details


There were some aspects of protocol that were disabled within chromium browsers (e.g. CDP listening) but there were other aspects that were not (e.g. Cypress events). This will disable all aspects of protocol in non chromium browsers

### Steps to test


Run tests in record mode with `DEBUG=cypress:*protocol*` and ensure that you don't see output in firefox but do in chrome.

### How has the user experience changed?


n/a

### PR Tasks


- [na] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [na] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
